src/ralph/reports/views.py

```diff
@@ -173,6 +232,106 @@ def prepare(self, model, dc=None):
             )
 
 
+class BaseRelationsReport(BaseReport):
+    template_name = 'reports/report_relations.html'
+    with_modes = True
+    links = False
+
+    def prepare(self, model, *args, **kwargs):
+        pass
+
+    def get_response(self, request, result):
+        data = tablib.Dataset(*result, headers=self.headers)
+        response = HttpResponse(
+            data.csv,
+            content_type='text/csv;charset=utf-8'
+        )
+        response['Content-Disposition'] = 'attachment;filename={}'.format(
+            self.filename
+        )
+        return response
+
+
+class AssetRelationsReport(BaseRelationsReport):
+    slug = 'asset-relations'
+    name = _('Asset - relations')
+    description = _('Asset list of information about the user, owner, model.')
+    filename = 'asset_relations.csv'
+    headers = ASSETS_COLUMNS
+
+    def get_assets_rows(self, model):
```

Comment:
there should be no option to generate report for ALL assets - it's useless; just maintain two separated reports for BO and DC and create separated columns for BO and DC.